templates/taxonomy-topic.php

```diff
@@ -35,6 +41,19 @@
 				</div>
 			<?php
 			}
+			?>
+				</div>
+				<div class="<?php UCF_FAQ_Config::add_athena_attr( 'col-4 mt-3' ); ?>">
+					<h3 class="<?php UCF_FAQ_Config::add_athena_attr( 'h5' ); ?>">TODO: Spotlight</h3>
```

Comment:
Whenever this and the related FAQs sections get updated, if these headings still exist, they'll need to be h2's